src/docs/development/platform-integration/add-to-app-android/add-flutter-screen.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Adding a Flutter screen to an Android app
+short-title: Add a Flutter screen
+description: Learn how to add a single Flutter screen to your existing Android app.
+---
+
+This guide describes how to add a single Flutter screen to an existing Android app. A Flutter
+screen can be added as a normal, opaque screen, or as a see-through, translucent screen. Both
+options are described in this guide.
+
+## Add a normal Flutter screen
+
+{% asset development/platform-integration/add-to-app-android/add-flutter-screen/add-single-flutter-screen_header.png class="mw-100" alt="Add Flutter Screen Header" %}
+
+### Step 1: Add Gradle dependencies
+
+TODO: update this with gradle instructions when the embedding is broken into multiple maven artifacts.
+
+### Step 2: Initialize Flutter
+
+Flutter requires that it be initialized before attempting to display any Flutter content. This can be done in one of three ways: 
+
+ * Use `FlutterApplication` as your app's `Application` class.
+
+ * Extend `FlutterApplication` with your own `Application` class.
+
+ * Manually initialize Flutter, yourself.
+
+#### Option 1: Use FlutterApplication
+
+To use `FlutterApplication` directly, simply declare it as your `Application` class in your
+`AndroidManifest.xml` file:
+
+```xml
+<application
+  android:name="io.flutter.app.FlutterApplication"
+  // other properties ellided
+  >
+  <!-- content ellided -->
+</application>
+```
+
+Now, as soon as your application is created, Flutter will initialize itself.
+
+#### Option 2: Extend FlutterApplication
+
+Extending `FlutterApplication` will automatically initialize Flutter when Android creates your application. First, create an `Application` class that extends `FlutterApplication`:
+
+```java
+public class MyApplication extends FlutterApplication {
+  // no implementation is necessary.
+}
+```
+
+Second, register your custom `Application` subclass in your `AndroidManifest.xml` file:
+
+```xml
+<application
+  android:name=".MyApplication"
+  // other properties ellided
```

Comment:
ellided => omitted

I love our rich array of vocabulary, but this would throw most native speakers, much less those who are not English proficient..